packages/expo-calendar/src/Calendar.ts

```diff
@@ -558,6 +558,149 @@ export type DaysOfTheWeek = {
 
 export { PermissionResponse, PermissionStatus, PermissionHookOptions };
 
+/**
+ * The result of presenting the calendar dialog.
+ * These constants map to [`EKEventEditViewAction`](https://developer.apple.com/documentation/eventkitui/ekeventeditviewaction) and [`EKEventViewAction`](https://developer.apple.com/documentation/eventkitui/ekeventviewaction).
+ * @platform ios
+ * */
+export type CalendarDialogResultActionsIos = 'done' | 'canceled' | 'deleted' | 'responded';
+
+/**
+ * The result of launching the calendar Activity.
+ * Android doesn't provide enough information to determine the user's action, so the result is always 'done'.
+ * The user may have cancelled the dialog, saved the event, or deleted it.
```

Comment:
We prefer American English over British English
```suggestion
 * The user may have canceled the dialog, saved the event, or deleted it.
```